src/components/LottieAnimations/index.tsx

```diff
@@ -1,79 +1,80 @@
+import type {LottieViewProps} from 'lottie-react-native';
 import colors from '@styles/theme/colors';
 import variables from '@styles/variables';
 import type DotLottieAnimation from './types';
 
 const DotLottieAnimations = {
     Abracadabra: {
-        file: require('@assets/animations/Abracadabra.lottie'),
+        file: require<LottieViewProps['source']>('@assets/animations/Abracadabra.lottie'),
```

Comment:
If you use `LottieViewProps['source']`, maybe we should add some type alias for it?